Makefile

```diff
@@ -34,6 +34,7 @@ mypy:
 
 .PHONY: test
 test:
+	@python tests/try_assert.py
 	pytest --cov=pydantic
```

Comment:
@samuelcolvin  I'm not sure which order is preferable here. Let me know if you'd like it changed.